decisions/0003-use-progressive-enhancement.md

```diff
@@ -0,0 +1,43 @@
+# 3. Use progressive enhancement
+
+Date: 2018-10-12
+
+## Status
+
+Accepted
+
+## Context
+
+The service manual clearly states that teams should use progressive enhancement
+when building services: https://www.gov.uk/service-manual/technology/using-progressive-enhancement
+
+This supports [point 12 of the service standard](https://www.gov.uk/service-manual/service-standard/create-a-service-thats-simple),
+which is about ensuring that all users can use the service successfully the
+first time they try - including users with accessibility needs.
+
+The service manual is also clear that [internal services should be held to the
+same standard as public-facing services](https://www.gov.uk/service-manual/design/services-for-government-users).
+
+Some of the services for prison and probation staff which have been built over
+the last couple of years are not progressively enhanced. Without JavaScript
+they display no content.
+
+Since these services are in a similar space to our work and have overlapping
+user bases with ours (although they are not the only existing services in this
+space), we have considered whether we should take a similar approach to them.
+
+## Decision
+
+We will use progressive enhancement for all our user-facing applications.
```

Comment:
I think it may be a good idea to document a higher level decision about whether we agree to follow the service standard (or not) then agree what aspects of this will be adhered to. I would imagine this goes beyond a technical decision and the product owner needs to be consulted.

Reply:
The decision on being assessed against the [service standard](https://www.gov.uk/service-manual/service-standard) or the [Technology Code of Practice](https://www.gov.uk/government/publications/technology-code-of-practice/technology-code-of-practice) isn't up to the team or HMPPS - it's part of the spend controls process which all our work goes through.

Point 2 of the TCOP is "Make things accessible", and the page with more detail on that point links back to the service standard. Progressive enhancement is the best starting point for making our services accessible, so there's no contradiction between the two standards we could be assessed against here. It's pretty clear that we're building a service rather than any other kind of technology, so we expect the service standard to apply in any case.